src/Tribe/Editor/Configuration.php

```diff
@@ -24,8 +64,12 @@ public function hook() {
 	 * @return array
 	 */
 	public function editor_config( $editor_config ) {
-		$tec = empty( $editor_config['events'] ) ? array() : $editor_config['events'];
+		$tec = empty( $editor_config['events'] ) ? [] : $editor_config['events'];
 		$editor_config['events'] = array_merge( (array) $tec, $this->localize() );
+
+		$post_objects = empty( $editor_config['post_objects'] ) ? [] : $editor_config['post_objects'];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 18 spaces but found 1 space

<sub>hash: 9181743cb1a86bea982f5cb70d916fb5</sub>